frontend/src/pages/Common/EmptyStates.js

```diff
@@ -0,0 +1,245 @@
+        />
+        <Typography variant="body1" className={classes.caption}>
+          {strings.common.no_notifications}
+        </Typography>
+      </div>
+    );
+  })
+);
+
+const Users = withTheme()(
+  withStyles(styles)(props => {
+    const { classes } = props;
+    return (
+      <Paper style={{ textAlign: "center" }}>
+        <img src="images-for-empty-state/users-table-empty-state.png" alt={strings.common.no_users} width="505vw" />
+        <Typography variant="subtitle1" className={classes.subtitle}>
+          {strings.common.no_users}
+        </Typography>
+        <Typography variant="caption" className={classes.caption}>
+          {strings.common.no_users_text}
+        </Typography>
+        <br />
+      </Paper>
+    );
+  })
+);
+
+const UserGroups = withTheme()(
+  withStyles(styles)(props => {
+    const { classes } = props;
+    return (
+      <Paper style={{ textAlign: "center" }}>
+        <img
+          src="images-for-empty-state/users-group-table-empty-state.png"
+          alt={strings.common.no_groups}
+          width="597vw"
+        />
+        <Typography variant="subtitle1" className={classes.subtitle}>
+          {strings.common.no_groups}
+        </Typography>
+        <Typography variant="caption" className={classes.caption}>
+          {strings.common.no_groups_text}
+        </Typography>
+        <br />
+      </Paper>
+    );
+  })
+);
+
+const NewOrganizations = withTheme()(
+  withStyles(styles)(props => {
+    const { classes } = props;
+    return (
+      <table style={{ alignItems: "center" }}>
+        <tbody>
+          <tr>
+            <td width="200vw">
+              <img
+                src="images-for-empty-state/organization-empty-state.png"
+                alt={strings.common.no_organizations}
+                width="150vw"
+              />
+            </td>
+            <td>
+              <Typography variant="body2" className={classes.caption}>
+                {strings.common.no_organizations}
+              </Typography>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    );
+  })
+);
+
+const ExistingNodes = withTheme()(
+  withStyles(styles)(props => {
+    const { classes } = props;
+    return (
+      <table style={{ alignItems: "center" }}>
+        <tbody>
+          <tr>
+            <td width="200vw">
+              <img
+                src="images-for-empty-state/nodes-for-orga-empty-state.png"
+                alt={strings.common.no_nodes}
+                width="150vw"
+              />
+            </td>
+            <td>
+              <Typography variant="body2" className={classes.caption}>
+                {strings.common.no_nodes}
+              </Typography>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    );
+  })
+);
+
+export {
+  Subproject,
+  ProjectBudget,
+  WorkflowItem,
+  SubprojectBudget,
+  Notification,
+  Users,
+  UserGroups,
+  NewOrganizations,
+  ExistingNodes
+};
```

Comment:
it kind of makes sense to have them all in one place, but i would still suggest to split them up in separate files, but in a dedicated folder maybe, i.e. `frontend/src/pages/Common/EmptyStates/`
and please note that the word "empty" should also be in the title of the component, i.e. `Subproject` should be `EmptySubprojectTable` since this is what is displayed

Reply:
additional thought: since each component has a very specific context (`Notification` will only be displayed on the notification page) it would make sense to put these components directly in the pages where they are needed